001-rfcs.md

```diff
@@ -0,0 +1,25 @@
+
+# RFCs
+
+We want to start using RFCs to discuss (and document) architectural decisions.
+
+**Discussion:** https://github.com/DoSomething/rfcs/pull/1
+
+## Problem
+
+Ideas tend to get stuck in one person's head by default (or lost in 1:1 conversations). As our team grows, we want to make it easy for everyone to take part in architectural discussions & see where we're thinking about taking our platform.
```

Comment:
I see this mainly referring to architectural discussions, but I know we've talked about using RFCs for how our team operates and other non-architectural processes. Would it make sense to update the wording here to show that the suggestions don't have to be architectural?

Reply:
Mmmm, good point! I tried re-phrasing this in 8baebe7.